File: app/src/main/java/ru/visualmath/android/api/model/Question.java
package ru.visualmath.android.api.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class Question implements Serializable {

    @SerializedName("_id")
    private String id;

    @SerializedName("question")
    private String title;

    @SerializedName("multiple")
    private boolean multiple;

    @SerializedName("answers")
    private List<String> answers;

    @SerializedName("isAnswerSymbolic")
    private boolean symbolic;

    public String getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public boolean isMultiple() {
        return multiple;
    }

    public List<String> getAnswers() {
        return new ArrayList<>(answers);
    }

    public boolean isSymbolic() {
        return symbolic;
    }
}

File: app/src/main/java/ru/visualmath/android/api/model/Question.kt
package ru.visualmath.android.api.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Question(

        @SerializedName("_id")
        val id: String = "",

        @SerializedName("question")
        val title: String = "",

        @SerializedName("multiple")
        val isMultiple: Boolean = false,

        @SerializedName("answers")
        val answers: List<String> = emptyList(),

        @SerializedName("isAnswerSymbolic")
        val isSymbolic: Boolean = false
) : Parcelable
